wcmc_components/lib/wcmc_components/filterable.rb

```diff
@@ -1,301 +1,84 @@
+require 'wcmc_components/filterable/attributes'
+require 'wcmc_components/filterable/csv_generator'
+require 'wcmc_components/filterable/parameters'
+require 'wcmc_components/filterable/query_object'
+require 'wcmc_components/filterable/serializer'
+
 module WcmcComponents
   module Filterable
-    def self.included(base)
-      base.extend ClassMethods
-    end
+    extend ActiveSupport::Concern
 
-    module InstanceMethods
-    end
+    included do
+      class_attribute :table_attributes, default: Attributes.new
 
-    module ClassMethods
-      # declare attrs you want to filter on
-      def table_attr(attr, options = {})
-        (@table_attrs ||= {})[attr] = options
-      end
+      delegate :form_attributes, :table_columns, :attributes_for_table, to: :table_attributes
 
-      def filters
-        table_attrs.select { |_k, v| v[:filter_on] } || {}
+      # table_page_path returns the 'show' path for the resource
+      def table_page_path
+        ''
```

Comment:
What does this do ? Return an empty string ?

Reply:
Yeah, this I assume was placeholder, but is updated in the edit branch to give the path to the record e.g. `countries/1`